src/cli/msig/index.ts

```diff
@@ -164,6 +168,177 @@ export default async function msig(args: string[]) {
+                    ownerPermission.required_auth.accounts.map((permissionWeight) => ({
+                        permission: {
+                            actor: permissionWeight.permission.actor.toString(),
+                            permission: permissionWeight.permission.permission.toString(),
+                        },
+                        weight: permissionWeight.weight.toNumber(),
+                    })),
+                    ownerPermission.required_auth.waits.map((waitWeight) => ({
+                        // eslint-disable-next-line camelcase
+                        wait_sec: waitWeight.wait_sec.toNumber(),
+                        weight: waitWeight.weight.toNumber(),
+                    }))
+                );
+                const activeAuthority = new Authority(
+                    activePermission.required_auth.threshold.toNumber(),
+                    activePermission.required_auth.keys.map((keyWeight) => ({
+                        key: keyWeight.key.toString(),
+                        weight: keyWeight.weight.toNumber(),
+                    })),
+                    activePermission.required_auth.accounts.map((permissionWeight) => ({
+                        permission: {
+                            actor: permissionWeight.permission.actor.toString(),
+                            permission: permissionWeight.permission.permission.toString(),
+                        },
+                        weight: permissionWeight.weight.toNumber(),
+                    })),
+                    activePermission.required_auth.waits.map((waitWeight) => ({
+                        // eslint-disable-next-line camelcase
+                        wait_sec: waitWeight.wait_sec.toNumber(),
+                        weight: waitWeight.weight.toNumber(),
+                    }))
+                );
+
+                activeAuthority.addCodePermission('vesting.tmy');
+                ownerAuthority.addCodePermission('vesting.tmy');
+
+                const actions = ['owner', 'active'].map((permission) => ({
+                    account: 'eosio',
+                    name: 'updateauth',
+                    authorization: [
+                        {
+                            actor: account,
+                            permission: permission,
+                        },
+                    ],
+                    data: {
+                        account: account,
+                        permission: permission,
+                        parent: permission === 'owner' ? '' : 'owner',
+                        auth: permission === 'owner' ? ownerAuthority : activeAuthority,
+                        // eslint-disable-next-line camelcase
+                        auth_parent: false,
+                    },
+                }));
+
+                proposalName = Name.from(`'${proposalName} + ${count}`);
+                console.log('proposalName', proposalName);
+
+                const proposalHash = await createProposal(
```

Comment:
lets create 1 proposal for all of the updateauth transactions for all accounts. not one per account